__tests__/nc-news.test.js

```diff
@@ -325,3 +324,29 @@ describe("204 - DELETE - /api/comments/:comment_id", () => {
       });
   });
 });
+describe("200 - GET -/api/users", () => {
+  it("should respond with an array of objects with the following properties: username, name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const users = response.body.users;
+        console.log(users);
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
+        });
+      });
+  });
+  it("should respond with 404 - not found - if no users found", () => {
+    return request(app)
+      .get("/api/user,z")
```

Comment:
I'm not sure what this is testing? isn't it just testing the route not found error handler is working?